lib/active_merchant/billing/gateways/fusebox.rb

```diff
@@ -129,7 +130,12 @@ def inquiry(fields)
       end
 
       # Storing a card is done via an authorize for $0.00 with the string "ID:" in the token request field
+      # NB: Some cards/banks return an error for $0 auth, so you have to use $1 auth and then reverse it.
       def store(creditcard, options = {})
+        authorize(0, creditcard, options.merge(:token_request => 'ID:'))
+      end
+
+      def store_and_auth(creditcard, options = {})
```

Comment:
This works for now, but I'm wondering if 2 different methods is the right solution for release to upstream ActiveMerchant.

I'm concerned because:
* Both methods do an auth, but only one is called `store_and_auth`
* Its unclear which I would want to use and when, if I'm coming to this new
* What if I felt the proper amount was a 1cent auth?

What about having one `store` method that uses a default amount of `0` but can be overridden with another amount in the `options` hash:

```rb
store(card) # Will auth for $0
store(card, auth_amount: 101) # Will auth for $1.01
```